src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,42 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name) {
+    this.name = name;
+    this.health = 100;
```

Comment:
```suggestion
  constructor(name, health = 100) {
    this.name = name;
    this.health = health;
```
you can pass health as a parameter, that let you change health in the future (just advice, optional for this task)